### PR TITLE
Skill - Swarp

### PR DESCRIPTION
**Pull request by JesterWizard**
I've started to get back into porting the existing skills. 

Skill effect: Swap the skill holder and an ally unit using rescue's range.

### CODE REVIEW

File: Data/SkillSys/SkillInfo.c [hunk 1]
```diff
@@ -3568,6 +3568,11 @@ const struct SkillInfo gSkillInfos[MAX_SKILL_NUM + 1] = {
     },
 #endif
 
+#if (defined(SID_Swarp) && COMMON_SKILL_VALID(SID_Swarp))
+    [SID_Swarp] = {
+        .desc = MSG_SKILL_Swarp,
```
Comment (MokhaLeee) [inline Data/SkillSys/SkillInfo.c:3573]: seems encounters some error when merging to master branch

Comment (JesterWizard) [reply Data/SkillSys/SkillInfo.c:3573]: Weird, Guthub says it's good to merge. Can you paste them here?

Comment (MokhaLeee) [reply Data/SkillSys/SkillInfo.c:3573]: I mean it is apparently the skill info is lost here, I'm not sure whther there are more mistakes elsewhere, maybe you'd better re-pull a new PR after testing

Comment (JesterWizard) [reply Data/SkillSys/SkillInfo.c:3573]: Ah it must have gotten removed accidently when I was merging the conflicts. I'll add it back in and review with my local

Comment (JesterWizard) [reply Data/SkillSys/SkillInfo.c:3573]: Okay all good

**Pull request merged**